### PR TITLE
Лабораторная работа 3

### PR DESCRIPTION
**Pull request by smolnikovaea**

### CODE REVIEW

File: Лабораторная работа 3/task 6.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+for i in range(len(list_numbersber)):
+    max_number = list_numbers[max_index]
+    current_number = list_numbers[i]
+    if current_number > list_numbers[max_index]:
+        max_index = i
+
+
+        last_index = list_numbers[-1]
+        list_numbers[-1] = list_numbers[max_index]
+        list_numbers[max_index] = last_index
```
Comment (EgorOb) [inline Лабораторная работа 3/task 6.py:14]:
```
last_index = list_numbers[-1]
list_numbers[-1] = list_numbers[max_index]
list_numbers[max_index] = last_index
```
Необходимо вынести за цикл

Comment (EgorOb) [reply Лабораторная работа 3/task 6.py:14]: И так же можно использовать особенность Python и его возможность множественного присваивания
`list_numbers[-1], list_numbers[max_index] = list_numbers[max_index], list_numbers[-1]`

File: Лабораторная работа 3/task 8.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+spend = 6000
+increase = 0.05
+
+month = 0  # количество месяцев, которое можно прожить
+loss = salary -(spend + (spend*(month*increase)))
+ostalos = money_capital + loss
+while ostalos >= 0:
+    ostalos = money_capital = loss
+    month += 1
+    loss = salary -(spend + (spend*(month*increase)))
```
Comment (EgorOb) [inline Лабораторная работа 3/task 8.py:12]: можно было бы переопределить spend, то loss определялся бы проще визуально
```
spend *= 1 + increase
loss = salary - spend
```

**Review by EgorOb — APPROVED** [review]
Зачтено.
